Source/Endless_Hotel/Anomaly/Object/ShelfBook/Anomaly_Object_ShelfBook.h

```diff
@@ -0,0 +1,61 @@
+// Copyright by 2025-2 WAP Game 2 team
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Anomaly/Object/Anomaly_Object_Base.h"
+#include "Components/SceneComponent.h"
+#include "Components/BoxComponent.h"
+#include "Components/StaticMeshComponent.h"
```

Comment:
```suggestion
#include "Components/SceneComponent.h"
#include "Components/BoxComponent.h"
#include "Components/StaticMeshComponent.h"
```
해당 부분들 헤더파일 아니고 cpp 파일로 옮겨주세요

Reply:
알겠습니다